devicetypes/smartthings/ezex-smart-sub-meter.src/ezex-smart-sub-meter.groovy

```diff
@@ -0,0 +1,116 @@
+/**
+ *  Copyright 2019 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+    definition (name: "eZEX Smart Submeter", namespace: "smartthings", author: "SmartThings") {
```

Comment:
This and https://github.com/SmartThingsCommunity/SmartThingsPublic/pull/3975 seem very similar.  What is the need for 2 different DTHs

Reply:
#3975  have capibility switch, bu this one not

Reply:
Got it, thanks.